webroot/js/app.js

```diff
@@ -387,9 +387,8 @@ export default class App extends Component {
     const bgLogo = { backgroundImage: `url(${smallLogo})` };
     const bgLogoLarge = { backgroundImage: `url(${largeLogo})` };
 
-    const tagList = !tags.length
-      ? null
-      : tags.map(
+    const tagList = (tags !== null && tags.length > 0)
```

Comment:
Now that I think about it, we could probably just do a check for `(!tags) ?..` and not worry about `tags.length` as the mapping will just stop if length=0.

Reply:
With this, it's explicit 🗡️

Reply:
While it is explicit, it will simplify the logic and make it easier to read. 🤔

Reply:
Did as you pleased, _now_ it *should* be done.